great_expectations/util.py

```diff
@@ -1673,7 +1674,7 @@ def get_context(
     ge_cloud_base_url: Optional[str] = None,
     ge_cloud_access_token: Optional[str] = None,
     ge_cloud_organization_id: Optional[str] = None,
-    ge_cloud_mode: bool = False,
+    ge_cloud_mode: bool = None,
```

Comment:
Can you change the docstring description and args to reflect the changes?

Reply:
yes 716abe064